src/__tests__/Display.spec.js

```diff
@@ -1,10 +1,83 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 
 import Display from '../components/Display/Display';
+import App from '../App';
+import Panel from '../components/Panel/Panel';
 
 describe('<Display />', () => {
-	it('renders without crashing', () => {
-		shallow(<Display />);
+	it('renders shallow without crashing', () => {
+	  shallow(<Display />);
 	});
-});
+	it('renders prop name in div', () => {
+	  const wrapper = shallow(<Display value="abcd" />);
+	  const finder = wrapper.find('div');
+  
+	  expect(finder.last().text()).toBe('abcd');
+	});
+	it('renders prop name in div', () => {
+	  const wrapper = shallow(<Display value="abcd" />);
+	  const finder = wrapper.find('.component-display');
+  
+	  expect(finder.last().text()).toBe('abcd');
+	});
+	it('renders depending on App state', () => {
+	  const wrapper = mount(
```

Comment:
nice job experimenting with mounting! you might find this article interesting https://blog.kentcdodds.com/why-i-never-use-shallow-rendering-c08851a68bb7